### PR TITLE
Efp 10 date change

### PR DESCRIPTION
**Pull request by typer177**
[EFP-10](https://codelation.atlassian.net/browse/EFP-10)

### CODE REVIEW

**Review by t-whitney — CHANGES_REQUESTED** [review]
It might be a good idea to have some sort of message or icon indicating the new check-in date has been saved

File: src/components/Dashboard/Orders.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ class Dashboard extends Component {
       other: ' ',
       pickup_name: '',
       checkout_at: ' ',
+      checkin_at: '',
```
[the remark this answers is not in the document]
Comment (typer177) [reply src/components/Dashboard/Orders.js:31]: fixed

File: src/components/Dashboard/Orders.js [hunk 2]
```diff
@@ -113,7 +115,31 @@ class Dashboard extends Component {
     )
   }
 
+  updateOrderDate = (event) => {
+    let orderObj = this.state.orderObj;
```
[the remark this answers is not in the document]
Comment (typer177) [reply src/components/Dashboard/Orders.js:119]: yeah, thanks for that. fixed

File: src/components/Dashboard/Orders.js [hunk 3]
```diff
@@ -113,7 +115,31 @@ class Dashboard extends Component {
...
+    const checkin = new Date(orderObj.checkin_at);
+    const [year, month, day] = [...event.target.value.split('-')];
+    checkin.setFullYear(year);
+    checkin.setMonth(month - 1);
+    checkin.setDate(day);
+    orderObj.checkin_at = checkin;
+    
+    this.setState({
```
[the remark this answers is not in the document]
Comment (typer177) [reply src/components/Dashboard/Orders.js:127]: From the looks of it, the code is setup to use redux reducers for actions (loading message, errors, general data), sagas are threads for api calls, and things like what is above are local.

**Comment by typer177** [comment]
Would it be helpful to let the user change the check-in date upon checking them in? Instead of having to check in for today and then clicking back in and changing the date?

This isn't something that happens very often, so it makes sense for there to be an extra step to change the date

**Review by t-whitney — APPROVED** [review]

**Pull request merged**